### PR TITLE
Walls and windows for Fiorina.

### PR DESCRIPTION
**Pull request by Cu1r**
## About the PR
Added 4 window types and 4 wall types for Fiorina Science Annex.

## Why / Balance
Fiorina need walls to keep the marines from discovering breathing is optional now and barotrauma doesn't exist.

## Media
walls

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

File: Resources/Textures/_RMC14/Structures/Walls/bone_resin.rsi/bone_resin.png
Comment (QuietlyWhisper) [inline Resources/Textures/_RMC14/Structures/Walls/bone_resin.rsi/bone_resin.png:1]: Don't think you need this image if I'm not mistaken

File: Resources/Textures/_RMC14/Structures/Walls/prison_rwall.rsi/hwall.png
Comment (QuietlyWhisper) [inline Resources/Textures/_RMC14/Structures/Walls/prison_rwall.rsi/hwall.png:1]: Don't think you need this image if I'm not mistaken

File: Resources/Textures/_RMC14/Structures/Walls/prison_rwall.rsi/rwall.png
Comment (QuietlyWhisper) [inline Resources/Textures/_RMC14/Structures/Walls/prison_rwall.rsi/rwall.png:1]: Don't think you need this image if I'm not mistaken

File: Resources/Textures/_RMC14/Structures/Walls/prison_wall.rsi/metal.png
Comment (QuietlyWhisper) [inline Resources/Textures/_RMC14/Structures/Walls/prison_wall.rsi/metal.png:1]: Don't think you need this image if I'm not mistaken

File: Resources/Prototypes/_RMC14/Entities/Structures/Windows/windows.yml
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Entities/Structures/Windows/windows.yml:1]: Can you add todo rmc14 to these comments

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]
I don't think the window sprites are correct here, they shouldn't be individual sprites

**Review by DrSmugleaf — DISMISSED** [review]
Thanks for the PR! We can fix the smoothing later

**Review by DrSmugleaf — APPROVED** [review]

**Pull request merged**